nuxt-frontend/tailwind.config.js

```diff
@@ -36,9 +36,11 @@ module.exports = {
           900: "#3d3d3d",
           87: "#000000DE",
           60: "#00000099",
+          darktone: "#121212",
```

Comment:
it's not a shade of black, so remove it from here and define new with a `deep-charcoal` name